lib/ronin/app/cli.rb

```diff
@@ -104,10 +104,18 @@ def run
         Dir.chdir(ROOT)
 
         begin
-          unless is_redis_running?
-            log_info "Starting Redis server ..."
-            pids << start_redis
-            sleep 1
+          if is_valkey_installed?
+            unless is_valkey_running?
+              log_info "Starting Valkey server ..."
+              pids << start_valkey
+              sleep 1
+            end
+          else
+            unless is_redis_running?
```

Comment:
These two statements can be combined into `elsif !is_redis_running?`.

Reply:
> These two statements can be combined into elsif !is_redis_running?.

Hmm but in that case if valkey is installed and running, but redis is not running it will start `redis-server` instead of skipping this step